internal/cri/server/container_list.go

```diff
@@ -58,19 +63,32 @@ func toCRIContainer(container containerstore.Container) *runtime.Container {
 	}
 }
 
-func (c *criService) normalizeContainerFilter(filter *runtime.ContainerFilter) {
```

Comment:
I think original code is working because shortcut id has multiple matches. so, containerd won't update id with full id in this case. that's why you don't see any thing. I don't think we should return error here. when you use one product and filter something, based on my experience, I don't see any error if there is no match result. it's just empty. in this case, I think it should be that same. 

If we have to return error, we should think about compatible issue because it's api behaviour change.